src/index.html

```diff
@@ -4,9 +4,39 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--day--start-thu calendar--month--length-29">
```

Comment:
to follow BEM namings
```suggestion
    <div class="calendar calendar--start-day--thu calendar--month-length--29">
```